ftw/noticeboard/types/category.py

```diff
@@ -0,0 +1,23 @@
+from ftw.noticeboard.interfaces import ICategory
+from plone.app.dexterity.behaviors.metadata import IBasic
+from plone.dexterity.content import Container
+from plone.supermodel import model
+from zope.interface import alsoProvides
+from zope.interface import implementer
+from zope.interface import implements
+from zope.schema.interfaces import ICollection
+
+
+@implementer(ICollection)
```

Comment:
What does this do?

Reply:
This does implement the ICollection behaviour.

Reply:
Why would we want that?

Reply:
@2e12 ?